api/login/login.controller.js

```diff
@@ -109,13 +109,18 @@ module.exports = {
       res.send("Unauthorized token");
     }
 
+    //use public and private key...
     const { access_token } = req.query;
     const payload = { access_token };
-    const token = jwt.sign(payload, process.env.ACCESS_TOKEN_SECRET, { expiresIn: '7d' });
-    const refreshtoken = 'Bearer ' + token;
-
-    // res.options.headers('authorization', 'Bearer ' + token);
 
+    const private_key= getMyPrivateKey();
```

Comment:
Let's have better method naming. :) `getSigningKey` or `getPrivateKey` would be better I think.